platform/admin/core/admin-core-migration-commands/src/test/java/org/codice/ddf/migration/commands/AbstractMigrationCommandTest.java

```diff
@@ -0,0 +1,93 @@
+package org.codice.ddf.migration.commands;
+
+import java.io.PrintStream;
+import java.nio.file.LinkOption;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+import org.codice.ddf.configuration.migration.ConfigurationMigrationService;
+import org.codice.ddf.migration.MigrationReport;
+import org.codice.ddf.security.common.Security;
+import org.fusesource.jansi.Ansi;
+import org.fusesource.jansi.Ansi.Attribute;
+import org.hamcrest.Matcher;
+import org.hamcrest.Matchers;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.rules.TemporaryFolder;
+import org.mockito.AdditionalAnswers;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+
+public class AbstractMigrationCommandTest {
+    protected final PrintStream CONSOLE = Mockito.mock(PrintStream.class);
+
+    protected final Path EXPORTED_PATH = Paths.get("test-exported");
+
+    protected final String EXPORTED_ARG = EXPORTED_PATH.toString();
+
+    protected final ConfigurationMigrationService SERVICE = Mockito.mock(
+            ConfigurationMigrationService.class);
+
+    protected final Security SECURITY = Mockito.mock(Security.class);
+
+    @Rule
+    public TemporaryFolder testFolder = new TemporaryFolder();
+
+    protected Path ROOT;
+
+    protected Path DDF_HOME;
+
+    @Before
+    public void baseBefore() throws Exception {
+        ROOT = testFolder.getRoot()
+                .toPath()
+                .toRealPath(LinkOption.NOFOLLOW_LINKS);
+        DDF_HOME = testFolder.newFolder("ddf")
+                .toPath()
+                .toRealPath(LinkOption.NOFOLLOW_LINKS);
+        DDF_HOME.resolve(EXPORTED_PATH)
+                .toFile()
+                .mkdirs();
+
+        Mockito.doAnswer(AdditionalAnswers.<MigrationReport, Callable<MigrationReport>>answer(code -> code.call()))
+                .when(SECURITY)
+                .runWithSubjectOrElevate(Mockito.notNull());
+
+        System.setProperty("ddf.home", DDF_HOME.toString());
+    }
+
+    protected <T extends MigrationCommand> T initCommand(T command) {
+        final T cmd = Mockito.spy(command);
+
+        Mockito.when(cmd.getConsole())
+                .thenReturn(CONSOLE);
+        return cmd;
+    }
+
+    protected void verifyConsoleOutput(Matcher<String> messageMatcher, Ansi.Color color) {
+        final ArgumentCaptor<String> argument = ArgumentCaptor.forClass(String.class);
+
+        Mockito.verify(CONSOLE, Mockito.times(2))
```

Comment:
Not a fan. I always find it less clear since it can be thought to be a local method unless you check the static imports. Got to say that has never been a feature of Java I liked.

Reply:
Your ide can help with this. I have all method types color coded (instance calls, static calls, etc). Also technically it could be a method in a super class that is inherited , which you also wouldn't be able to find in the local class. 

IMO the overhead of discovering and remember what is a static method can be beneficial if it reduces the overhead of verbosity by an equal or greater amount. Meaning, after seeing 100 Mockito.verifies I'm pretty certain it's going to be mockito.verifies. 

Using something like quick documentation or just CMD/ctrl clicking in your IDE will never be replaceable. 

To your point though, other languages like go specifically require every non local call to be qualified  exactly for this reason.